Detectors/MUON/MCH/Simulation/include/MCHSimulation/Response.h

```diff
@@ -23,20 +23,24 @@ namespace mch
 class Response
 {
  public:
-  Response() = default;
-
+  Response(int station = 0): mStation(station) { init(); };
```

Comment:
I would drop the init function. The init _is_ the constructor.

Reply:
Ok, done. now everything inside the constructor.